common/installer/installer.go

```diff
@@ -17,10 +17,20 @@ type K8sInstaller interface {
 	Uninstall() string
 }
 
+// archOldNameMap keeps the mapping of architecture new name to old name mapping
+var archOldNameMap = map[string]string{
+	"amd64": "x86-64",
+}
+
 // NewInstaller will return a new installer
 func NewInstaller(ctx context.Context, osDist, arch, k8sVersion string, downloader BundleDownloader) (K8sInstaller, error) {
+	bundleArchName := arch
+	// replacing the arch name to old name to match with the bundle name
+	if _, exists := archOldNameMap[arch]; exists {
+		bundleArchName = archOldNameMap[arch]
+	}
 	// normalizing os image name and adding arch
-	osArch := strings.ReplaceAll(osDist, " ", "_") + "_" + arch
+	osArch := strings.ReplaceAll(osDist, " ", "_") + "_" + bundleArchName
```

Comment:
This still won't fix the problem as the regex are different for os_detector and host_registration. We need to remove the "LTS" from osDist here. Small comparison test 🙂 : https://go.dev/play/p/QQ4IjESzFDo

Reply:
I have added "LTS" in test cases and it's working fine.  We check for the os distribution using regex: `Ubuntu_20.04.*_x86-64`. So, this won't be a problem.

Reply:
Ohh then it is perfect 👍